### ISSUE TITLE
Ionic CSS attributes are deprecated

### ISSUE THREAD

**Issue by marcmayans** [eakoriakin/ionic-selectable#227]
Replace:
'<div margin>' and '<ion-list no-margin>'

With:
'<div class="ion-margin">' and '<ion-list class="ion-no-margin">'

**Comment by gr99t** [comment]
I guess he forgot to quote as code so the <> tags get lost.

```
[DEPRECATED][CSS] Ionic CSS attributes are deprecated.
Replace:
'<ion-virtual-scroll no-margin>'

With:
'<ion-virtual-scroll class="ion-no-margin">'
```

That is at least what I still get in two projects using the latest 4.5.1 :thinking:

**Comment by edy-ap** [comment]
thanks, i will check

**Comment by edy-ap** [comment]
can you show an example in stackblitz, please

**Comment by gr99t** [comment]
Sure, check [this](https://stackblitz.com/edit/ionic-v4-angular-tabs-q99wdy?file=src%2Fapp%2Ftab1%2Ftab1.page.html) out

**Comment by edy-ap** [comment]
thanks, it fixed in release 4.5.2, is now available on npm
